docs/migrating.storybook.mdx

```diff
@@ -1,91 +1,293 @@
+```
+
+Other ways to verify the migration is done:
+
+-   you will see the `--wave` CSS variables in dev tools;
+-   if your OS switches between light and dark schemes, the UI will switch to the dark scheme too (you can enforce it in dev tools);
+
+In case it doesn't work, reach #ask-wave Slack channel for help.
+
+## 4️⃣ Enforce light scheme
+
+Wave v2 comes with automatic dark scheme enabled by default. That means the color of the UI will reflect the OS color preferences of the user.
+
+Most likely, your app will need adjustments to make the dark scheme readable.
+Consider adding a task to your backlog, and disable the dark scheme by wrapping your application root with the `LightScheme` until you are ready:
+
+```tsx
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App.tsx';
+import { ClassicColors, LightScheme } from '@freenow/wave'; // blue primary color
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+    <React.StrictMode>
+        <ClassicColors />
+        <LightScheme>
+            <App />
+        </LightScheme>
+    </React.StrictMode>
+);
+```
+
+## 5️⃣ Polish the code
+
+Codemods leave several artifacts that require manual processing. Your code will work fine without polishing, but it will be repetitive.
+
+### Migrate to semantic CSS variables
+
+Find all `--wave-b-color` usages. Those are CSS variables pointing to theme colors, like `Color` object we had before.
+The better way to work with colors is to use _semantic CSS variables_, they start with `--wave-s-color`.
+Consider the role the color fulfils in your use case, and pick the matching variable from several semantic groups:
+
+-   _foreground_ for texts and icons;
+-   _background_ for background of elements and surfaces;
+-   _border_;
+-   _shadow_.
+
+The list of semantic variables is available at [the colors page](?path=/docs/essentials-colors--docs),
+but we recommend using a typed `getSemanticValue` helper instead of hardcoding variable names (they are long).
+
+<CssVariablesMigrationTable />
+
+### Set the default page background to Wave's
+
+Wave has several CSS variables for page background color because of the dark/light schemes.
+We automatically set the background color to the `body`, but you probably have it set too.
+If so, delete your manual page background setting or set it to `getSemanticValue('background-page-default')`.
+
+### Clean `inverted` props migration
+
+If you used `inverted` prop extensively, you will see that each component is now wrapped into `<InvertedColorScheme>` component.
+This is not required, you can wrap multiple components at once and avoid duplication (`InvertedColorScheme` is `Box` with a special class name):
+
+<InvertedPropMigrationTable />
+
+### Review `ref` usages
+
+We changed the place where `ref` is attached to the form components `Input`, `Password`, `PhoneInput`, etc.
```

Comment:
In the case of the `PhoneInput` we are not forwarding the ref, we only had to change it since it has imperative behaviour on it's internal `Input` component. The change should be transparent to our users so I think we don't need to mention it.
```suggestion
We changed the place where the `ref` is attached to the form components `Input` and `Password`.
```